### PR TITLE
Migrated to Dedicated SQL pool

### PR DESCRIPTION
**Pull request by cmendible**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Upgraded template to Dedicated SQL pool (formerly SQL DW)

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@julieMSFT - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@julieMSFT - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/metadata.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "type": "QuickStart",
-  "itemDisplayName": "SQL Server SQL Data Warehouse with Transparent Encryption.",
-  "description": "Creates a SQL Server and a SQL Data Warehouse with Transparent Data Encryption.",
-  "summary": "This template creates a SQL Server and a SQL Data Warehouse with Transparent Data Encryption.",
+  "itemDisplayName": "Dedicated SQL pool with Transparent Encryption.",
+  "description": "Creates a SQL Server and a Dedicated SQL pool (formerly SQL DW) with Transparent Data Encryption.",
+  "summary": "This template creates a SQL Server and a Dedicated SQL pool (formerly SQL DW) with Transparent Data Encryption.",
   "githubUsername": "cmendible",
   "docOwner": "julieMSFT",
```
Comment (WilliamDAssafMSFT) [inline quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/metadata.json:8]:
```suggestion
  "docOwner": "WilliamDAssafMSFT",
```
Please go ahead and accept this suggestion to also update this codeowner, Julie is no longer with MSFT.

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/metadata.json (hunk 1 above)
Comment (cmendible) [reply quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/metadata.json]: Done!

**Comment by azure-quickstarts** [comment]
@WilliamDAssafMSFT - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/azuredeploy.json [hunk 2]
```diff
@@ -38,11 +38,19 @@
         "description": "Enable/Disable Transparent Data Encryption"
       }
     },
-    "serviceLevelObjective": {
+    "capacity": {
+      "type": "int",
+      "metadata": {
+        "description": "The Elastic Pool DTU or nomber of vcore."
```
Comment (WilliamDAssafMSFT) [inline quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/azuredeploy.json:44]: @cmendible Please review this. This should not be labeled as an elastic pool DTU or number of vcores, but rather a string that is the DW size, for example, DW400c.

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/azuredeploy.json (hunk 2 above)
Comment (cmendible) [reply quickstarts/microsoft.sql/sql-data-warehouse-transparent-encryption-create/azuredeploy.json]: Fixed based on the values provided in this table: https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-service-capacity-limits#workload-management and the description displayed in the Azure portal.

**Comment by azure-quickstarts** [comment]
@WilliamDAssafMSFT - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**